Makefile

```diff
@@ -133,6 +137,13 @@ $(BINDIR)/client-gen:
 		--clientset-path github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/ \
 		--go-header-file vendor/github.com/kubernetes/repo-infra/verify/boilerplate/boilerplate.go.txt
 	  # the previous three directories will be changed from kubernetes to apimachinery in the future
+# generate lister
```

Comment:
I always use tabs in makefiles, but apparently that's not standard either. If you really want it, I'll put up with bright neon pink warnings.